app/views/articles/subscriptions.html.erb

```diff
@@ -0,0 +1,65 @@
+<% title "Subscriptions #{@article.title}" %>
+
+<% data = [
```

Comment:
Should this PR be a draft containing sample data like this?

Reply:
Very good point! Thanks :)